main.js

```diff
@@ -66,17 +87,55 @@ function setAPPListeners () {
  * Opens new window with index.html(Jitsi Meet is loaded in iframe there).
  */
 function createJitsiMeetWindow () {
-    jitsiMeetWindow = new BrowserWindow(jitsiMeetWindowOptions);
-    jitsiMeetWindow.loadURL(indexURL);
+  jitsiMeetWindow = new BrowserWindow(jitsiMeetWindowOptions);
+  jitsiMeetWindow.loadURL(indexURL);
+  microWindow = new BrowserWindow(microWindowOptions);
```

Comment:
I'd like us to test creating the window lazily. Having the window always there will make it "visible" to some window managers on Linux, for example.